### PR TITLE
Fixes #28485 - Fix PG::AmbiguousColumn error on tasks search

### PR DESCRIPTION
**Pull request by adamruzicka**
The original error was
PG::UndefinedColumn: ERROR: column foreman_tasks_tasks.duration does not exist |
LINE 1: ...) as duration FROM "foreman_tasks_tasks"

### CODE REVIEW

**Comment by adamruzicka** [comment]
CC @ezr-ondrej in the end, using a has for ordering broke sorting by duration, which is not a real column

File: app/controllers/foreman_tasks/api/tasks_controller.rb [hunk 1]
```diff
@@ -238,9 +238,10 @@ def pagination_scope(scope, search_params)
 
       def ordering_scope(scope, ordering_params)
         sort_by = ordering_params[:sort_by] || 'started_at'
+        sort_by = 'foreman_tasks_tasks.' + sort_by if sort_by == 'started_at'
```
Comment (ezr-ondrej) [inline app/controllers/foreman_tasks/api/tasks_controller.rb]: this raises a question if other columns aren't going to be ambigous :(

Comment (ares) [reply app/controllers/foreman_tasks/api/tasks_controller.rb]: as of today the sort_by is possible only by started_at and duration. We can improve in future by prefixing everything except for duration quite easily if the need be, so I'm fine taking this as is.

**Comment by ezr-ondrej** [comment]
@ares feel free to merge, my concern is not a blocker.

**Comment by ares** [comment]
Thanks, merging!

**Pull request merged**